test/bats/test.bats

```diff
@@ -81,6 +86,22 @@ teardown() {
   run kubectl apply -f ${BATS_TESTS_DIR}/bad/bad_ns.yaml
   assert_match 'denied the request' "$output"
   assert_failure
+
+  run kubectl apply -f ${BATS_TESTS_DIR}/constraints/all_ns_must_have_gatekeeper-dryrun.yaml
+  assert_success
+
+  observedGeneration=$(kubectl get k8srequiredlabels.constraints.gatekeeper.sh ns-must-have-gk -o json | jq '.status.byPod[0].observedGeneration')
+  generation=$(kubectl get k8srequiredlabels.constraints.gatekeeper.sh ns-must-have-gk -o json | jq '.metadata.generation')
+  run wait_for_process $WAIT_TIME $SLEEP_TIME "[[ $observedGeneration = $generation ]]"
```

Comment:
Because these variables are set above, they wont change to new values as part of `wait_for_process`. We could make the observedGeneration == generation logic into a function and run wait_for_process over that.

Reply:
thanks, updated